docker/netbox/netbox.env

```diff
@@ -29,9 +30,14 @@ REDIS_INSECURE_SKIP_TLS_VERIFY=false
 REDIS_PASSWORD=nwa
 REDIS_SSL=false
 RELEASE_CHECK_URL=https://api.github.com/repos/netbox-community/netbox/releases
-SECRET_KEY=r8OwDznj!!dci#P9ghmRfdu1Ysxm0AiPeDCQhKE+N_rClfWNjN
-SKIP_SUPERUSER=false
+SECRET_KEY="NNV%+&Bv8=3ILWDDMu1^Uc&wUUrTfc*1@QNN5L)^xl@NF_jJ1@"
```

Comment:
I take it this secret key is not actually used in the demo environment?

Reply:
I don't know much about this setup but I can see that in the `docker-compose.yaml` file there is:

`  env_file: ./docker/netbox/netbox.env
`